resources/translations/fi.edn

```diff
@@ -213,12 +213,12 @@
                          :message "Hyvä %1,\nSinut on lisätty jäseneksi hakemukseen %2.\nKatsele hakemusta: %3"}
           :member-invited {:subject "Kutsu osallistumaan hakemukseen"
                            :message "Hei,\nTämä sähköpostiosoite (%1) on kutsuttu osallistumaan hakemukseen.\nOsallistumislinkki: %2"}}
-  :footer "CSC - IT Center for Science"
-  :form {:accepted-licenses "Hyväksynyt käyttöehdot"
+  :footer "CSC – Tieteen tietotekniikan keskus"
+  :form {:accepted-licenses "Käyttöehdot hyväksytty"
          :actions "Toiminnot"
-         :add-comments-not-shown-to-applicant "Lisää kommentti (ei näy hakijalle)"
-         :add-comments-shown-to-applicant "Lisää kommentti (näkyy hakijalle)"
-         :alert-disabled-resources "Seuraavia resursseja ei voi enää hakea."
+         :add-comments-not-shown-to-applicant "Kommentoi (ei näy hakijalle)"
+         :add-comments-shown-to-applicant "Kommentoi (näkyy hakijalle)"
+         :alert-disabled-resources "Resurssit eivät ole haettavissa."
```

Comment:
Same as English that "enää" is important

Reply:
Done.